contrib/windows-cmake/CMakeLists.txt

```
@@ -87,8 +114,7 @@ if(NOT HWLOC_SKIP_TOOLS)
     foreach(tool IN ITEMS ${TOOLS})
         add_executable(${tool}
             ${TOPDIR}/utils/hwloc/${tool}.c)
-        target_include_directories(${tool} PRIVATE ${TOPDIR}/include ${CMAKE_CURRENT_BINARY_DIR}/include)
-        target_link_libraries(${tool} PRIVATE libhwloc)
+        target_link_libraries(${tool} PRIVATE hwloc::hwloc)
```

```suggestion
        target_link_libraries(${tool} PRIVATE libhwloc)
```
there is no reason to use rerouting targets in the project itself.

---

This is to avoid repeating properties target_include_directories(), instead using the object that propagates those properties

---

`libhwloc` was also a target at the point of review (probably renamed to just `hwloc`) with the same usage requirements. You don't need to use `hwloc::hwloc`